prow/cluster/jenkins_deployment.yaml

```diff
@@ -25,9 +25,10 @@ spec:
     spec:
       containers:
       - name: jenkins-operator
-        image: gcr.io/k8s-prow/jenkins-operator:0.39
+        image: gcr.io/k8s-prow/jenkins-operator:.1
```

Comment:
@kargakis right now ...
```
$ prow/bump.sh jenkins-operator
program: jenkins-operator
old version: 
new version: .1
$ prow/bump.sh jenkins
program: jenkins
old version: 0.39
new version: 0.40
```

One of these updates the deployment, one gets the Makefile?